### PR TITLE
RHDM-382: Loading popup displayed indefinitely after deleting project

### PR DESCRIPTION
**Pull request by mbarkley**
Related: https://github.com/kiegroup/appformer/pull/185

### CODE REVIEW

**Comment by mbarkley** [comment]
@manstis Assuming @jhrcek verifies this fix, I will have to cherry-pick this to master. I am open to suggestions if you think this close-able scheduler should be moved somewhere else more reusable.

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/ProjectScreen.java [hunk 1]
```diff
@@ -119,7 +120,8 @@ public ProjectScreen(final View view,
                          final Classifier assetClassifier,
                          final Event<AssetDetailEvent> assetDetailEvent,
                          final BusyIndicatorView busyIndicatorView,
-                         final ProjectController projectController) {
+                         final ProjectController projectController,
+                         final Scheduler scheduler) {
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/ProjectScreen.java]: Shouldn't `CloseableSchedulerWrapper` be injected here?

Comment (mbarkley) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/ProjectScreen.java]: `CloseableSchedulerWrapper` is a sub-type of `Scheduler`, so there's no need to reference the exact type.

**Comment by jhrcek** [comment]
Jenkins execute full downstream build

**Comment by manstis** [comment]
@mbarkley ```kie-wb-common``` seems a good a place as any.

**Pull request merged**